sdk/maps/azure-maps-search/azure/maps/search/aio/_search_client_async.py

```diff
@@ -0,0 +1,131 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+# pylint: disable=unused-import,ungrouped-imports
+from typing import TYPE_CHECKING
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.exceptions import HttpResponseError
+from azure.core.credentials import AzureKeyCredential
+from ._base_client_async import AsyncMapsSearchClientBase
+from .._generated.models import BatchRequest, SearchAddressBatchResult
+
+if TYPE_CHECKING:
+    from typing import Any, List
+    from azure.core.credentials_async import AsyncTokenCredential
+    from azure.core.polling import LROPoller, AsyncLROPoller
+
+
+# By default, use the latest supported API version
+class MapsSearchClient(AsyncMapsSearchClientBase):
+    """Azure Maps Search REST APIs.
+
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :param client_id: Specifies which account is intended for usage in conjunction with the Azure AD security model.
+     It represents a unique ID for the Azure Maps account and can be retrieved from the Azure Maps management
+     plane Account API. To use Azure AD security in Azure Maps see the following
+     `articles <https://aka.ms/amauthdetails>`_ for guidance.
+    :type client_id: str
+    :param str base_url: Service URL
+    :keyword int polling_interval: Default waiting time between two polls for
+     LRO operations if no Retry-After header is present.
+    :keyword api_version:
+            The API version of the service to use for requests. It defaults to the latest service version.
+            Setting to an older version may result in reduced feature compatibility.
+    :paramtype api_version: str or ~azure.ai.translation.document.MapsSearchApiVersion
+    """
+    def __init__(
+        self,
+        credential, # type: Union[AzureKeyCredential, AsyncTokenCredential]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        super().__init__(
+            credential=credential, **kwargs
+        )
+
+    @distributed_trace_async
+    def begin_fuzzy_search_batch(
+        self,
+        batch_request,  # type: "BatchRequest"
```

Comment:
I'm not doing an API review here - but we should probably remove this `BatchRequest` object and just accept a list of query strings. The batch result could potentially also be simplified in a similar manner.

Reply:
It is now accepting list of query strings. thanks.